pkg/policies/engine/rego/rego.go

```diff
@@ -145,7 +151,51 @@ func (r *Rego) Verify(ctx context.Context, policy *engine.Policy, input []byte,
 		}
 	}
 
-	return violations, nil
+	return &engine.EvaluationResult{
+		Violations: violations,
+		Skipped:    false, // best effort
+		Message:    "",
+	}, nil
+}
+
+func parseResultRule(res rego.ResultSet, policy *engine.Policy) (*engine.EvaluationResult, error) {
```

Comment:
could you elaborate here in a comment the logic used to calculate skipped/result?

Unit testing this method would be useful too.

Reply:
Merge doesn't happen at this layer. It's done in the `policies` package, since it's what the crafter uses to verify materials.